scripts/pre_commit_linter.py

```diff
@@ -1537,15 +1537,15 @@ def _check_sorted_dependencies(all_files, parsed_js_files):
                     if sorted_imports != function_args:
                         failed = True
                         print (
-                            'Please ensure that %s in %s, the injected '
-                            'dependecies should be in the following manner:'
+                            'Please ensure that in %s in %s, the injected '
```

Comment:
Okay, I am re-iterating the earlier comment here: `Please ensure that in %s in file %s, ....`. Please change this here and below.

Reply:
Done.